iOS_SDK/OneSignalSDK/Source/OSStateSynchronizer.m

```diff
@@ -266,14 +254,12 @@ - (void)sendLocation:(os_last_location *)lastLocation
                appId:(NSString *)appId
          networkType:(NSNumber *)networkType
      backgroundState:(BOOL)background {
-    let pushStateSyncronizer = [self getPushStateSynchronizer];
-    let emailStateSyncronizer = [self getEmailStateSynchronizer];
-    
+    let stateSyncronizer = [self getStateSynchronizers];
+    // Begin constructing the request for the external id update
```

Comment:
This comment isn't correct, this is a location update. Recommend removing instead of updating comment as the method name already describes the request about to be made.